### PR TITLE
revert the removal of the null param for authentication scheme and ad…

### PR DESCRIPTION
**Pull request by jennyf19**
…d the tests to rel/v2

removing broke some tests, so adding it back.

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
 name: .NET Core
 
-on: [push]
+on:
+  push:
+    branches:
+      - "master"
+      - "rel/v2"
```
Comment (jmprieur) [inline .github/workflows/dotnetcore.yml:7]: ❤️

File: src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs [hunk 2]
```diff
@@ -641,7 +647,7 @@ private IConfidentialClientApplication BuildConfidentialClientApplication(Merged
                     else
                     {
                         string? sessionKey = tokenAcquisitionOptions!.LongRunningWebApiSessionKey;
-                        if (sessionKey == TokenAcquisitionOptions.LongRunningWebApiSessionKeyAuto)
+                        if (sessionKey == Abstractions.AcquireTokenOptions.LongRunningWebApiSessionKeyAuto)
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs:650]: Is it something we missed?

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19

File: src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs (hunk 2 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs:650]: i don't think so, it was a suggestion from VS

**Pull request merged**